### ISSUE TITLE
[Arcs] Loyal Cards on other Spends

### ISSUE THREAD

**Issue by ledergamesmatt** [LederCards/cards#27]
We ruled the Pacifist Act 2 Objective is exclusively Psionic resources and that Loyal Cards should have a consistency errata to say: "You may spend any resources _like they are_ <Loyal Resource>..."

This also applies to Cloud Cities. Can I use, say, a Relic to build a Cloud City on a Fuel planet if I have Loyal Pilots? I suppose my question is fundamentally, if Cloud Cities says that it "costs" 1 resource to build a Cloud City, are you "spending" resources when you pay "costs"?

- [ ] Apply ruling to Loyal Engineers BC01
- [ ] Apply ruling to Loyal Pilots BC07
- [ ] Apply ruling to Loyal Marines BC15
- [ ] Apply ruling to Loyal Empaths BC19
- [ ] Apply ruling to Loyal Keepers BC21
- [ ] Apply ruling to Loyal Engineers F605
- [ ] Apply ruling to Loyal Pilots F606
- [ ] Apply ruling to Loyal Marines F607
- [ ] Apply ruling to Loyal Empaths F608
- [ ] Apply ruling to Loyal Keepers F609

**Comment by jdyearsley** [comment]
Looking back at the fuller implications of this ruling, I'm actually inclined to reverse it and bite the bullet on letting Loyal Empaths be _extremely_ good in a Partisan-Pacifist matchup, since people can always steal it. Otherwise, as this issue suggests, it will require a lot of rewording for consistency as well as other places where, intuitively, you should just be able to use Loyal cards to do things with any resources. 

I'd like to assess whether there are wider-reaching implications before reversing this, but that's the direction I'm leaning.

**Comment by jdyearsley** [comment]
Yes, I'm still okay with biting the bullet—I will reverse the Pacifist ruling.
